### ISSUE TITLE
TeraVM Regression | AGW Internally triggered Downlink ICMP packets on GTP to UE

### ISSUE THREAD

**Issue by arun-magma** [magma/magma#7749]
We are finding Downlink ICMP packets on GTP towards UE, which is causing issues especially while trying to test idle mode scenarios as it keeps always the UE tunnels in active state.
It seems to be originating from AGW itself , We like to know how to stop this flow and curious to understand the purpose of this ICMP pkts.

[Snap of pcap where we see ICMP from AGW to UE IP]

![image](https://user-images.githubusercontent.com/70725754/123310710-3d853b00-d4db-11eb-9c44-b3ec38e97eeb.png)
[gx_gy_combined_05_210624_094429-1.zip](https://github.com/magma/magma/files/6711255/gx_gy_combined_05_210624_094429-1.zip)

**Comment by arun-magma** [comment]
cc: @ulaskozat @uri200

**Comment by arun-magma** [comment]
@vaishaliCI

**Comment by ardzoht** [comment]
Hi @arun-magma we discovered that `monitord` service was enabled on the teravm gateway configuration, this service is made mainly for CPE deployments (which don't usually go into idle mode) to get latency of the active CPEs connected to the gateway. On TeraVM however it's weird that this was enabled.

I updated the configuration to disable this service, so this issue should be resolved, feel free to open it again if more issues arise after this. Thanks!
